x-pack/test/alerting_api_integration/common/lib/alert_utils.ts

```diff
@@ -176,38 +184,41 @@ export class AlertUtils {
     if (this.user) {
       request = request.auth(this.user.username, this.user.password);
     }
-    const response = await request.send({
-      enabled: true,
-      name: 'abc',
-      schedule: { interval: '1m' },
-      throttle: '1m',
-      tags: [],
-      alertTypeId: 'test.always-firing',
-      consumer: 'bar',
-      params: {
-        index: ES_TEST_INDEX_NAME,
-        reference,
-      },
-      actions: [
-        {
-          group: 'default',
-          id: this.indexRecordActionId,
-          params: {
-            index: ES_TEST_INDEX_NAME,
-            reference,
-            message:
-              'instanceContextValue: {{context.instanceContextValue}}, instanceStateValue: {{state.instanceStateValue}}',
-          },
-        },
-      ],
-      ...overwrites,
-    });
+    const alertBody = getDefaultAlwaysFiringAlertData(reference, actionId);
+    const response = await request.send({ ...alertBody, ...overwrites });
     if (response.statusCode === 200) {
       objRemover.add(this.space.id, response.body.id, 'alert');
     }
     return response;
   }
 
+  public async updateAlwaysFiringAction({
+    alertId,
+    actionId,
+    reference,
+    user,
+    overwrites = {},
+  }: UpdateAlwaysFiringAction) {
+    actionId = actionId || this.indexRecordActionId;
```

Comment:
nit. worth inlining as a default value, no?

Reply:
ya, prolly, assuming I can default using a `this` reference.  The fn I copied this from has two of these, I'll fix those also ...

Reply:
well dang, I forgot to fix this one before merging to master, but it's a nit.  I'm sure @gmmorris will flag again next time around!